src/app/(proper_react)/redesign/(public)/PlansTable.module.scss

```diff
@@ -207,11 +207,168 @@
+    align-items: center;
+    justify-content: start;
+    gap: $spacing-lg;
+    background-color: $color-white;
+    border-radius: $border-radius-md;
+    padding: $spacing-2xl $spacing-lg;
+
+    hr {
+      border-style: none;
+      border-top: 1px solid $color-grey-20;
+      width: 100%;
+    }
+
+    .head {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-sm;
+
+      h3 {
+        font: $text-title-xs;
+        font-weight: 600;
+
+        b {
+          color: $color-purple-70;
+        }
+      }
+    }
+
+    .priceSection {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-lg;
+
+      .cost {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: $spacing-md;
+
+        .price {
+          font: $text-title-sm;
+        }
+
+        .total {
+          display: block;
+          color: $color-grey-40;
+
+          em {
+            font-weight: 700;
+            font-style: normal;
+          }
+        }
+      }
+
+      .cta {
+        font-weight: 700;
+      }
+
+      .reassurance {
+        font: $text-body-sm;
+        font-weight: 400;
+      }
+    }
+
+    .featuresSection {
+      display: flex;
+      flex-direction: column;
+      gap: $spacing-md;
+      padding-inline: $spacing-md;
+
+      h4 {
+        font: $text-title-3xs;
+      }
+
+      .featureList {
+        list-style-type: none;
+        display: flex;
+        flex-direction: column;
+        align-items: start;
+        justify-content: center;
+        gap: $spacing-lg;
+        margin: 0;
+        padding: 0;
+
+        li {
+          display: flex;
+          align-items: start;
+          gap: $spacing-sm;
+
+          .inclusionIcon {
+            margin: $spacing-xs;
+            flex: 1 0 auto;
+          }
+
+          &.included .inclusionIcon {
+            color: $color-green-90;
+          }
+
+          &.notIncluded .inclusionIcon {
+            color: $color-red-60;
+          }
+
+          button {
+            flex: 1 0 auto;
+          }
+        }
+      }
+    }
+
+    &.plusCard {
+      position: relative;
+      border: 4px solid $color-purple-70;
+
+      .badge {
+        position: absolute;
+        top: 0;
+        left: 50%;
+        transform: translateY(-50%) translateX(-50%);
+        background-color: $color-purple-70;
+        color: $color-white;
+        border-radius: $border-radius-md;
+        font: $text-body-sm;
+        font-weight: 600;
+        padding: $spacing-sm $spacing-md;
+
+        // CSS uppercasing is not reliable for some locales, so limit it to
+        // English, where it is. See
+        // https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues
+        [lang="en"] &,
+        [lang="en-US"] &,
+        [lang="en-CA"] &,
+        [lang="en-GB"] & {
+          text-transform: uppercase;
+        }
```

Comment:
Heh, that's a good point, as that made me realise that I should also add the flip side: that we should avoid uppercasing the source text, as suggested in that link, because it will cause screen readers to spell it out like an acronym. So I'm not sure yet whether this is a good long-term solution to enforce across the codebase, but I at least also called out the flip side now: 1045db0b0 and d7bf3d8c6.